libs/template/src/lib/let/let.directive.ts

```diff
@@ -303,14 +294,25 @@ export class LetDirective<U> implements OnInit, OnDestroy {
   set rxLetSuspense(
     templateRef: TemplateRef<LetViewContext<U | undefined | null> | null>
   ) {
-    this.templateManager.addTemplateRef('rxSuspense', templateRef);
+    this.templateManager.addTemplateRef('S', templateRef);
   }
 
-  private subscription: Unsubscribable = new Subscription();
-  private readonly templateManager: TemplateManager<
-    LetViewContext<U | undefined | null>,
-    RxTemplateName
-  >;
+  private _renderObserver: NextObserver<U>;
+  @Input()
+  set rxLetRenderCallback(callback: NextObserver<U>) {
+    this._renderObserver = callback;
+  }
+
+  // We use defer here as the as otherwise the the `@Output` decorator subscribes earlier than the arnderAware property
+  // is assigned
+  @Output() readonly rendered = defer(() => this.renderAware.rendered$.pipe(
+    filter(({ kind }) => this.templateManager.hasTemplateRef(kind)),
+    map(({ value }) => value as U),
```

Comment:
If I'm not mistaken the `value` is already inferred as `U` (at least my IDE says so). Please check it out, maybe this `as` clause won't be necessary ;)

Reply:
ty, looks good